inrupt/pod-browser
# Apps 1123 account dropdown title fix

## Description
This PR fixes the recently changed profile which breaks the account dropdown UI element ([1123](https://inrupt.atlassian.net/browse/APPS-1123)).

## Changes
 - Change the session test helper to have configurable profile documents
 - Some minor cleanup in `container/index.jsx` just because I was in that file and couldn't help myself
 - Also some minor cleanup in `usePodIrisFromWebId`
 - Changed `fetchProfile` to accommodate the new profile workflow

## Testing
 - Log into Pod Browser via another provider
 - Use `https://openid.dev-next.inrupt.com`
 - Verify you see your account name or nickname in the account menu
 
## Commit checklist


## Notes
I couldn't resist and made some minor cleanup to a couple files here. I realize that some may find this less than ideal so I'll gladly remove that code if preferred. Otherwise, I'd like to just sneak in some minor clean up.

## Screenshots/captures
![profile-menu](https://user-images.githubusercontent.com/35955/155410355-66b37536-7e30-415f-9db6-5769e6b66789.png)
